packages/zero-client/src/client/zero.ts

```diff
@@ -1960,6 +1974,8 @@ export async function createSocket(
   let queriesPatch: Map<string, UpQueriesPatchOp> | undefined =
     await queriesPatchP;
 
+  const activeClients = await aliveClientsManager?.getActiveClients();
```

Comment:
is there a distinction between active and alive?

Reply:
No, I stopped using alive and forgot to rename this